addons/web/static/src/js/views/form_common.js

```diff
@@ -752,6 +752,8 @@ var AbstractField = FormWidget.extend(FieldInterface, {
      * value is similar to false in OpenERP.
      */
     is_false: function() {
+        if (Object.prototype.toString.call(this.get('value')) === "[object Array]")
+                return this.get('value')[0] === false;
```

Comment:
BTW, Instead of handling here, would you please check why we set val || [false, flase] in set_value, is there any reason to set [false, false], what if we set val || false